src/System.Windows.Forms/src/System/Windows/Forms/Button.cs

```diff
@@ -399,5 +387,43 @@ protected override void WndProc(ref Message m)
                     break;
             }
         }
+
+        protected override AccessibleObject CreateAccessibilityInstance()
+           => new ButtonAccessibleObject(this);
+
+        [ComVisible(true)]
+        internal class ButtonAccessibleObject : ControlAccessibleObject
+        {
+            internal ButtonAccessibleObject(Button owner) : base(owner)
+            {
+            }
+
+            public override string Name
+            {
+                get => base.Name;
+                set => base.Name = value;
+            }
```

Comment:
Please remove it as this does not bring anything in comparison with `base.Name`.